src/import/sst_service.rs

```diff
@@ -106,6 +113,10 @@ impl<Router: RaftStoreRouter> ImportSst for ImportSSTService<Router> {
         )
     }
 
+    /// Ingest the file by sending a raft command to raftstore.
+    ///
+    /// If the ingest fails due to region not found or epoch does not match, the
```

Comment:
ingest -> ingestion

Reply:
due to region not found or epoch does not match -> because the region is not found or the epoch does not match